packages/@lwc/engine-core/src/framework/api.ts

```diff
@@ -159,8 +172,7 @@ function s(
     const { renderMode, shadowMode } = vmBeingRendered;
 
     if (renderMode === RenderMode.Light) {
-        sc(children);
-        return children;
+        return fr(data.key, children);
```

Comment:
can we do something similar for the slot children in Synthetic (line 177)?

I'm thinking about replacing the slot children with just the fragment. in that way we could completely remove `sc` (and the code associated with it). I'm also thinking that in such way, `pathChildren` will always go through the static diff, and patch fragment is the only one that will need to go through the slow path.